### ISSUE TITLE
requires a peer of unibeautify@>= x.x.x but none is installed

### ISSUE THREAD

**Issue by lassik** [Unibeautify/unibeautify-cli#24]
Unfortunately, I still haven't managed to actually format a file using the cli :-/ It doesn't manage to find the formatters I installed, or I didn't manage to install them properly.

```
$ sudo npm install --global unibeautify-cli
/usr/local/bin/unibeautify -> /usr/local/lib/node_modules/unibeautify-cli/dist/cli.js
+ unibeautify-cli@0.2.1
added 3 packages from 3 contributors and updated 2 packages in 5.195s
```

```
$ sudo npm install --global @unibeautify/beautifier-prettydiff
npm WARN @unibeautify/beautifier-prettydiff@0.5.3 requires a peer of unibeautify@>= 0.9.1 but none is installed. You must install peer dependencies yourself.
npm WARN @unibeautify/beautifier-prettydiff@0.5.3 requires a peer of prettydiff2@^2.2.7 but none is installed. You must install peer dependencies yourself.

+ @unibeautify/beautifier-prettydiff@0.5.3
updated 1 package in 3.976s
```

```
$ sudo npm install --global @unibeautify/beautifier-clang-format
npm WARN @unibeautify/beautifier-clang-format@0.2.0 requires a peer of unibeautify@>= 0.15.0 but none is installed. You must install peer dependencies yourself.

+ @unibeautify/beautifier-clang-format@0.2.0
updated 1 package in 1.48s
```

```
$ echo | unibeautify --language JavaScript
(node:18459) UnhandledPromiseRejectionWarning: Error: Beautifiers not found for Language: JavaScript
    at Unibeautify.beautify (/usr/local/lib/node_modules/unibeautify-cli/node_modules/unibeautify/dist/src/beautifier.js:69:35)
    at Socket.process.stdin.on (/usr/local/lib/node_modules/unibeautify-cli/dist/cli.js:43:25)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at Pipe.onread (net.js:635:20)
(node:18459) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:18459) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

```
$ echo | unibeautify --language C
(node:18541) UnhandledPromiseRejectionWarning: Error: Beautifiers not found for Language: C
    at Unibeautify.beautify (/usr/local/lib/node_modules/unibeautify-cli/node_modules/unibeautify/dist/src/beautifier.js:69:35)
    at Socket.process.stdin.on (/usr/local/lib/node_modules/unibeautify-cli/dist/cli.js:43:25)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at Pipe.onread (net.js:635:20)
(node:18541) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:18541) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Running on MacOS.

Is there something special about those `@` packages?

**Comment by szeck87** [comment]
Yea, it's the scoped package names.  [find-global-packages](https://github.com/chrisdickinson/find-global-packages) does not find them, nor does any other global package finder.  If you would like to try a working version out, pull the [ci-fixes](https://github.com/Unibeautify/unibeautify-cli/tree/ci-fixes) branch and install that.  It's not pretty right now, I'm using Node's spawn process to run `npm ls -g --depth=0 --json` and retrieving them that way.  @Glavin001 is going to look into better ways.

I have it working with `eslint` and `@unibeautify/beautifier-eslint` installed globally and running
```
unibeautify -l JavaScript --config-json '{"JavaScript": {"beautifiers": ["ESLint"],"quotes": "double"}}' <<< "const test = 'test';"
```
which outputs `const test = "test";` as I expect.

**Comment by szeck87** [comment]
Closed by #28
